tethys_cli/db_commands.py

```diff
@@ -0,0 +1,144 @@
+"""
+********************************************************************************
+* Name: db_commands.py
+* Author: Scott Christensen
+* Created On: 2019
+* Copyright: (c) Brigham Young University 2014
+* License: BSD 2-Clause
+********************************************************************************
+"""
+from pathlib import Path
+
+from django.conf import settings
+
+from tethys_cli.cli_helpers import get_manage_path, run_process, load_apps
+from tethys_apps.utilities import get_tethys_home_dir
+
+
+def add_db_parser(subparsers):
+    # DB COMMANDS
+    db_parser = subparsers.add_parser('db', help='Tethys Database Server utility commands.')
+
+    # Setup db commands
+    db_parser.add_argument('command',
+                           help='DB command to run.',
+                           choices=list(DB_COMMANDS.keys()))
+    db_parser.add_argument('-d', '--database', dest='db_alias',
+                           help="Name of the database options from settings.py to use.")
+    db_parser.add_argument('-n', '--username', dest='username',
+                           help="Name of super user to add to database when creating.")
```

Comment:
The help for the --username and --superuser-name is the same.

Reply:
Sorry, my comment on the help text for the db command wasn't clear. The help text for --username and --superusername both say that they are for creating superusers. Are both of those superusers?

Reply:
@sdc50 can you look at this? It’s in the docs now too.